concepticondata/conceptlists.tsv

```diff
@@ -316,3 +316,4 @@ Hartmann-2013-162	Hartmann, Iren and Haspelmath, Martin and Taylor, Bradley	2013
 Lundgren-2020-275	Lundgren, Olof	2020		275	areal	English	The Tupí–Guaraní languages Omagua and Kokama and their closest relative Tupinambá	https://lup.lub.lu.se/student-papers/search/publication/9007022	Lundgren2020		This 275-item list underlies the result of cognate detection in the Tupí–Guaraní languages Omagua and Kokama and their closest relative Tupinambá. The initial lexical data was drawn from the Tupí–Guaraní Comparative Lexical Dataset ([Chousou-Polydouri et al. 2019](:bib:ChousouPolydouri2019)). The cognate data were then used for the phonological reconstruction of their most recent common ancestor, Proto-Omagua–Kokama–Tupinambá.	89-97	
 Ruhlen-1998-36	Ruhlen, Merritt	1998		36	basic, areal	English	Yeniseian and Na-Dene	https://www.pnas.org/content/pnas/95/23/13994.full.pdf	Ruhlen1998		The basic concepts in this list serve to compare Proto-Yeniseian and Proto-Na-Dene cognates which are not widely shared by other language families. The author argues that this large amount of cognates of basic vocabulary can only be explained by a common ancestor language.	13995-13996	
 Lee-2015-246	Lee, Sean	2015		246	basic, areal	English	Koreanic	https://doi.org/10.1371/journal.pone.0128448	Lee2015		The basic concepts in this list are selected to &quot;quantitatively describe the evolutionary characteristics&quot; (of Koreanic).	e0128448	
+Urban-2016-160	Urban, Matthias	2016		160	basic	English	Global	http://doab-test.huma-num.fr/handle/20.500.12657/23584	Urban2016		This list of 160 nominal items was compiled in order to compare morphological complexity across 78 languages of different language families. The concepts were organized into four semantic categories relating to nature, artifacts, the body and temporal concepts (plus some miscellaneous words). The list is not directly based on another, although Brown 1999 and [Buck 1949](:ref:Buck-1949-1110) provided some inspiration.	571-572	
```

Comment:
```
The list is not directly based on another
```
rather:
```
The list is not directly based on another concept list
```
English doesn't allow to use another with out "one" or some filling word, and this is more explicit.